app/models/nonbillable_account.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class NonbillableAccount < Account
+  before_validation :set_owner, :set_description, :set_created_by, :set_account_number, :set_expries_at
+
+  # Since this account can be used by anyone, we only need one in the system
+  # so this class method should be used to access it.
+  def self.account
```

Comment:
I'm not sure how I feel about this name - how about
```suggestion
  def self.singleton_instance
```
I'm not attached to that, just think `account` is already over-used in the code